src/index.ts

```diff
@@ -0,0 +1,4 @@
+export type { VueDriftWidgetConfig, DriftWidget } from './types'
+export { createDrift } from './install'
+export { loadScript } from './utils'
```

Comment:
why export this helper?

Reply:
because createDrift can only be used with app.load. loadscript on the contrary can be used on a component level

Reply:
why would you call it the implementations assumes just setting the config to a new value and the script will load automatically...

Reply:
yeah, but again, createDrift ( now useDriftWidgetPlugin ) is used for the app.use() method in the main.ts. loadScript can be used to initialize the widget wherever you want, without the necessity to load the widget globally